packages/xo-web/src/common/intl/messages.js

```diff
@@ -1545,6 +1545,18 @@ const messages = {
   destroyTasksModalTitle: 'Destroy task{nTasks, plural, one {} other {s}}',
   destroyTasksModalMessage:
     'Are you sure you want to destroy {nTasks, number} task{nTasks, plural, one {} other {s}}?',
+  forgetHostFromSrModalTitle: 'Forget host',
+  forgetHostFromSrModalMessage:
+    'Are you sure you want to forget this host? This will disconnect the SR from the host by removing the link between the SR and the host (PBD).',
+  forgetHostsModalTitle: 'Forget host{nPbds, plural, one {} other {s}}',
+  forgetHostsModalMessage:
+    'Are you sure you want to forget {nPbds, number} host{nPbds, plural, one {} other {s}}? This will disconnect the SR from these hosts by removing the links between the SR and the hosts (PBDs)',
+  forgetSrModalTitle: 'Forget SR',
+  forgetSrModalMessage:
+    'Are you sure you want to forget this SR? This will disconnect the host from the SR by removing the link between the SR and the host (PBD).',
+  forgetSrsModalTitle: 'Forget SR{nPbds, plural, one {} other {s}}',
+  forgetSrsModalMessage:
```

Comment:
Let's keep all the labels consistent and call them `forget...from...ModalMessage`.